doc/site/zh-cn/guides/thread-primer/ipv6-addressing.md

```diff
@@ -0,0 +1,231 @@
+# IPv6 寻址
+
+让我们来看一下 Thread 如何识别网络中的每个设备，以及设备间用何种类型的地址进行相互通信。
+
+关键术语：在本入门教程中，术语“接口（interface）”用于标识网络内 Thread 设备的端点。通常，单个 Thread 设备具有单个 Thread 接口。
```

Comment:
Leave the Markdown callout as 

Key Point: <translated text>

Sorry, I forgot to mention this in the guidelines.  These will render on openthread.io based on the callout type, which is the term before the colon in Markdown format.  (The HTML ones are coded as `<aside class="note">`, for example... all the HTML should remain untranslated there).

So anything starting with these callout types shouldn't have it translated, just the text after the callout type.

Note:
Caution:
Warning: 
Key Term:
Key Point:

Reply:
🆗